dev_guide/templates.adoc

```diff
@@ -208,58 +165,6 @@ the following command:
 $ oc edit template <template>
 ----
 
-[[defining-template-parameters]]
-
-=== Defining Template Parameters
-
-Templates allow you to define parameters which take on a value. That value is
-then substituted whereever the parameter is referenced. References can be
-defined in any text field in the objects list field. The value can be set
-explicitly or it can be generated by OpenShift.
-
-Each parameter describes a variable and the variable value which can be
-referenced in any text field in the `objects` list field. The value can be set
-explicitly or it can be generated by the OpenShift.
-
-You can assign an explicit value to the parameter using the `value` field:
-
-----
-"parameters": [
-  {
-    "name": "USERNAME",
-    "description": "The user name for Joe",
-    "value": "joe"
-  }
-],
-----
-
-To generate a value using pseudo regular expression syntax you have to specify
-the `generate` field and set it to 'expression'. Then use the `from` field to
-specify the pattern that would be used to generate the value:
-
-----
-"parameters": [
-  {
-    "name": "PASSWORD",
-    "description": "The random user password",
-    "generate": "expression",
-    "from": "[a-zA-Z0-9]{12}"
-  }
-],
-----
-
-In example above, we generate 12 character long random password consisting of
-all upper and lowercase alphabet letters and numbers.
-
-The syntax used in above example is not full regular expression syntax. However you can use `\w`, `\d` and `\a` modifiers:
-
-* `[\w]{10}` produces 10 alphabet characters, excluding the numbers. This is equal to `[a-zA-Z]{10}`.
```

Comment:
This doesn't work as expected - https://github.com/openshift/origin/issues/3463 - left out for now.